### DISCUSSION TITLE
NO ACCESSIBILITY [OSX]

### DISCUSSION THREAD

**Discussion by rickhard04** [rsjaffe/MIDI2LR#575] [Problems]
Hello Everyone,
Ii have big issue form some days on my Mac Mini M1 with Monterey and latest release of Midi2lr.
I don't know what happened but I lost the permissions on the "Accessibility" section, Midi2LR has disappeared.
It is still present in the "Automation" section, it is present in the plugins inside LIGHTROOM, it is active and functional, but the program (MIDI2Lr) does not start.
It starts and closes immediately, without even opening the control window. 
I have deleted the plugin and the program several times, reinstalled, I uninstalled Lightroom and reinstalled by the beginning but the problem remains. The program does not start.
I also tried to create new catalogs, but the problem is on all catalogs ..

Someone can help me solve?

**Comment by rsjaffe** [comment]
I'm a little confused by the comment that it is in "automation" but not in "accessibility". Did you do what is described in the FAQ [After update, keyboard shortcuts don't work or crash application (MacOS only)](https://github.com/rsjaffe/MIDI2LR/wiki)?

**Reply by rsjaffe** [reply]
Remove the app from automation if it is present there. Make sure it is gone from all the panels in the MacOS privacy permissions list, and don't add it back unless MIDI2LR requests it.

**Reply by rickhard04** [reply]
ALSO My Friends Have The Plugin In Automation (See Image) But The Problem Is That My Friend Have Also Midi2LR In Accessibility, while I dont Have It.
And The Automatic Message for Have The Permission Never Appears.

![immagine](https://user-images.githubusercontent.com/95610843/144997599-7e23c606-f734-4628-87d7-fe509fc7e435.png)
